### PR TITLE
Add fixed navbar on scroll-up

### PR DESCRIPTION
**Pull request by jfranciscosousa**

### CODE REVIEW

**Comment by taniaprocha** [comment]
💣 

screenshot 2018-12-12 at 10 35 05

**Comment by jfranciscosousa** [comment]
I fixed this and forgot to push and now it's not on my home desktop :man_facepalming: :man_facepalming: I'll re-do it later

**Comment by jfranciscosousa** [comment]
Basically I'm missing a breakpoint to not display nothing on desktop

**Comment by jfranciscosousa** [comment]
@taniaprocha fixed

**Comment by taniaprocha** [comment]
💣 only works on mobile

**Comment by jfranciscosousa** [comment]
Fixed

File: src/components/Navbar/index.css [hunk 1]
```diff
@@ -1,6 +1,11 @@
 @import "../../styles/all.css";
 
 $modal-height: 402px;
+$navbar-background: color(white alpha(98%));
+$navbar-height: 68px;
+$navbar-height-mobile: 44px;
```
Comment (taniaprocha) [inline src/components/Navbar/index.css]: And you should confirm this height with Inês

Comment (jfranciscosousa) [reply src/components/Navbar/index.css]: These are browser side pixels, I measured manually.

Comment (jfranciscosousa) [reply src/components/Navbar/index.css]: They depend on some values but I don't think it's productive to use `calc` on everything.

Comment (jfranciscosousa) [reply src/components/Navbar/index.css]: As things are very much likely to change in the future.

File: src/components/Navbar/index.css [hunk 2]
```diff
@@ -1,6 +1,11 @@
 @import "../../styles/all.css";
 
 $modal-height: 402px;
+$navbar-background: color(white alpha(98%));
+$navbar-height: 68px;
```
[the remark this answers is not in the document]
Comment (jfranciscosousa) [reply src/components/Navbar/index.css]: Nope

[the remark this answers is not in the document]
Comment (taniaprocha) [reply src/components/Navbar/index.css]: Without padding you can change to `$navbar-height: 84px`

**Pull request merged**